CP01CNSM/CP01CNSM_R00009_ingest.csv

```diff
@@ -1,47 +1,47 @@
-mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl35/PCO2W*/SAMI*.txt,CP01CNSM-MFD35-05-PCO2WB000,recovered_inst,Expected,
-mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl35/phsen2/*.phsen2.log,CP01CNSM-MFD35-06-PHSEND000,recovered_host,Expected,
-mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl35/PHSEN*/SAMI_P0099_*.txt ,CP01CNSM-MFD35-06-PHSEND000,recovered_inst,Expected,
-mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl37/syslog/*.syslog.log,CP01CNSM-MFD37-00-DCLENG000,recovered_host,Expected,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/cpm3/syslog/cpm_status.txt,CP01CNSM-MFC31-00-CPMENG000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl36/syslog/*.syslog.log,CP01CNSM-MFD35-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.adcpt_acfgm.dcl.pd8.adcpt_acfgm_dcl_pd8_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl36/adcp/*adcp*.log,CP01CNSM-MFD35-01-ADCPTF000,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl36/ADCP*/*RDI*.000,CP01CNSM-MFD35-01-ADCPTF000,recovered_inst,Available,
+mi.dataset.driver.presf_abc.dcl.presf_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl36/presf/*.presf.log,CP01CNSM-MFD35-02-PRESFB000,recovered_host,Available,
+mi.dataset.driver.presf_abc.presf_abc_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl36/PRESF*/PRESF*.hex,CP01CNSM-MFD35-02-PRESFB000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl36/velpt2/*velpt*.log,CP01CNSM-MFD35-04-VELPTA000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl36/VELPT*/*.aqd,CP01CNSM-MFD35-04-VELPTA000,recovered_inst,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl36/pco2w/*.pco2w.log,CP01CNSM-MFD35-05-PCO2WB000,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl36/PCO2W*/SAMI*.txt,CP01CNSM-MFD35-05-PCO2WB000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/cg_data/dcl36/phsen2/*.phsen2.log,CP01CNSM-MFD35-06-PHSEND000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00009/instruments/dcl36/PHSEN*/SAMI_*.txt,CP01CNSM-MFD35-06-PHSEND000,recovered_inst,Available,

```

Comment:
Fixed MFN PHSEN filemask so that it could find data (removed space and made it more generic)